worker/build.rs

```diff
@@ -110,62 +104,49 @@ fn main() {
         println!("cargo:rustc-link-lib=dylib=c++");
         println!("cargo:rustc-link-lib=dylib=c++abi");
     }
-    #[cfg(target_os = "windows")]
-    {
-        if !std::path::Path::new("worker/out/msys/bin/make.exe").exists() {
-            let python = if let Ok(python) = env::var("PYTHON") {
-                python
-            } else if Command::new("where")
-                .arg("python3")
-                .status()
-                .expect("Failed to start")
-                .success()
-            {
-                "python3".to_string()
-            } else {
-                "python".to_string()
-            };
-
-            let dir = format!("{}/msys", mediasoup_out_dir.replace('\\', "/"));
-
-            if !Command::new(python)
-                .arg("scripts\\getmake.py")
-                .arg("--dir")
-                .arg(dir.clone())
-                .status()
-                .expect("Failed to start")
-                .success()
-            {
-                panic!("Failed to install MSYS/make")
-            }
-
-            env::set_var(
-                "PATH",
-                format!("{}\\bin;{}", dir, env::var("PATH").unwrap()),
-            );
-        }
 
-        env::set_var(
-            "PATH",
-            format!(
-                "{}\\worker\\out\\msys\\bin;{}",
-                env::current_dir()
-                    .unwrap()
-                    .into_os_string()
-                    .into_string()
-                    .unwrap(),
-                env::var("PATH").unwrap()
-            ),
-        );
+    // Install Python invoke package in custom folder
+    let pip_invoke_dir = format!("{out_dir}/pip_invoke");
+    let invoke_version = "2.2.0";
+    let python = env::var("PYTHON").unwrap_or("python3".to_string());
+    let mut pythonpath = if env::var("PYTHONPATH").is_ok() {
+        let original_pythonpath = env::var("PYTHONPATH").unwrap();
+        format!("{pip_invoke_dir}:{original_pythonpath}")
+    } else {
+        pip_invoke_dir.clone()
+    };
+
+    // Force ";" in PYTHONPATH on Windows
+    if cfg!(target_os = "windows") {
+        pythonpath = pythonpath.replace(':', ";");
+    }
+
+    if !Command::new(&python)
+        .arg("-m")
+        .arg("pip")
+        .arg("install")
+        .arg("--upgrade")
+        .arg(format!("--target={pip_invoke_dir}"))
```

Comment:
Shouldn't this be two separate arguments instead?:
```suggestion
        .arg("--target")
        .arg(pip_invoke_dir)
```